app/features/quiz/QuizSession/QuizAnswer/logic.js

```diff
@@ -315,6 +307,7 @@ export const recordAnswerLogic = createLogic({
       if (wrapUp.active) {
         dispatch(quiz.session.wrapUp.decrement());
       }
+      const isFinalQuestion = selectIsFinalQuestion(getState());
```

Comment:
It seems this move is necessary for this to work - if starting wrap-up with > 10 reviews in queue, the wrapUp count is **equal** to the queue length and so `wrapUp.count === 0` only once the decrement happens (line 308)